app/Modules/Person/Actions/DemographicsUpdate.php

```diff
@@ -0,0 +1,138 @@
+<?php
```

Comment:
This file is *very* similar to `ProfileUpdate.php`. Do we need both? Can they be refactored?